### ISSUE TITLE
Is the oracle procedure body displayed correctly?

### ISSUE THREAD

**Issue by Kazun3500** [dbeaver/dbeaver#12334]
#### System information: 
- Operating system (distribution) and version Linux Ubuntu 20.04
- DBeaver version - 21.0.2.202104042101

#### Connection specification:
- Database name and version - Oracle
- Driver name - default

#### Describe the problem you're observing:
I often use online service for unwrapping oracle packages - https://www.codecrete.net/UnwrapIt/ .
And I got the following problem using dbeaver and this service. As I understand it, dbeaver gets the body of the procedure with the following query and concatenating the received lines by newline character.
```
 select text from all_source where name = 'PACKAGE' 
    and type = 'PACKAGE BODY'
    order by line;
```
Part of example:
![image](https://user-images.githubusercontent.com/13629272/116890167-ff377200-ac35-11eb-8ee9-dfe20d1f4f7b.png)

https://www.codecrete.net/UnwrapIt/ can`t unwrap this. If i remove newlines added by dbeaver - all is ok, site unwraps package.

Where is an error - in Dbeaver or in site backend?

In another DB gui client (products from jetbrains) - package body concatenating by empty space ''

**Comment by LonwoLonwo** [comment]
Hello @Kazun3500 
Thanks for the bug report.

**Comment by Kazun3500** [comment]
Did you reproduce problem? Or example ddl is necessary?

**Comment by LonwoLonwo** [comment]
@Kazun3500 I reproduced the problem, thanks.

**Comment by LonwoLonwo** [comment]
Fixed in 21.0.5
Will be available [here](https://dbeaver.io/) at 2021-05-17

For testers: can be tested in SYS schema (ex: DBMSOBJG2 package, "Body" tab)

**Comment by uslss** [comment]
verified
